packages/core/src/rules/enableDeletionProtectionOnDynamoDBTables/doc.md

```diff
@@ -0,0 +1,15 @@
+# Enable Deletion Protection On DynamoDB Tables
+
+## Why ?
+
+Enabling this property for tables helps ensure that tables do not get accidentally deleted during regular table management operations by your administrators. This will help prevent disruption to your normal business operations.
+
+## How to fix ?
+
+In the AWS console: to change this setting, go to the table’s Additional settings, navigate to the Deletion Protection panel and select Enable delete protection.
```

Comment:
What about CDK or serverless framework? Can you add a link to the dedicated docs on DynamoDB?